common/culture/cultures/wc_tharian.txt

```diff
@@ -6,14 +6,19 @@
 	language = language_low_common
 	martial_custom = martial_custom_male_only
 	traditions = {
+		tradition_winter_warriors
+		tradition_practiced_pirates
```

Comment:
The Pirates tradition was reworked in b6f5e1278. 

>  tradition_practiced_pirates_name:0 "Might is Right"
 tradition_practiced_pirates_desc:0 "What need is there for diplomacy when we can simply take what is rightfully ours?"

Now it allows cultures to continue raiding over land even if they feudalise. However, only exceptions can raid overseas as defined in `00_rules,txt` and `wc_heritage.txt`.

Reply:
> Now it allows cultures to continue raiding over land even if they feudalise. However, only exceptions can raid overseas as defined in `00_rules,txt` and `wc_heritage.txt`.

It would be better to create a new tradition, but centaurs may unlock raiding by religion if it's the only point.

Reply:
I double checked and you're right about faith enabled raiding with `theradras_worship`. I'll replace `tradition_practiced_pirates` with `tradition_talent_acquisition` to reflect the subservient relationship of Snobolds towards their Magnataur masters.